docs/index.html

```diff
@@ -188,38 +189,38 @@ <h2 id="resampling">Convolution Resampling</h2>
             <a href="http://www.imagemagick.org/Usage/filter/#filter">
                 the convolution-based</a> method.
         <p>
-            "Convolution" means that for the each pixel of the final image
+            "Convolution" means that for each pixel of the final image
             we compute some area ("window")
             of the source image using weights.
-            The size of the area and weights depend on chosen filter.
+            The size of the area and weights depend on the chosen filter.
             There are many filters.
-            Most common of them are:
+            The most common of which are:
         <ul>
-            <li><code>Bilinear</code> — high-efficient filter
+            <li><code>Bilinear</code> — high-efficiency filter
                 with the small window and a blurry result;</li>
             <li><code>Bicubic</code> — high-quality filter with medium window;</li>
-            <li><code>Lanczos</code> — allows trade a quality over performance
+            <li><code>Lanczos</code> — allows a trade off of quality over performance
```

Comment:
I think that "trade off of quality" can be regarded as "sell quality, buy performance". But the actual meaning is opposite.